ui/apps/platform/src/Containers/MainPage/Navigation/NavigationSidebar.tsx

```diff
@@ -103,6 +89,17 @@ function getNavDescriptions(isFeatureFlagEnabled: IsFeatureFlagEnabled): NavDesc
                   type: 'separator',
                   key: 'following-node-cves',
               },
+
+              {
+                  type: 'link',
+                  content: 'Platform CVEs',
+                  path: vulnerabilitiesPlatformCvesPath,
+                  routeKey: 'vulnerabilities/platform-cves',
+              },
+              {
+                  type: 'separator',
+                  key: 'following-workload-cves',
```

Comment:
Understandable copy-paste that can wait until the names stop moving.

Reply:
😵‍💫